### ISSUE TITLE
SSL connection to Redis

### ISSUE THREAD

**Issue by rasanentimo** [lae/ansible-role-netbox#72]
Add option to enable SSL for Redis. This could be achieved by adding line ```netbox_redis_ssl_enabled: false``` on ```defaults/main.yml``` and modifying the redis section in ```templates/configuration.py.j2``` 
```
REDIS = {
    'HOST': '{{ netbox_redis_host }}',
    'PORT': '{{ netbox_redis_port }}',
    'PASSWORD': '{{ netbox_redis_password }}',
    'DATABASE': '{{ netbox_redis_database }}',
    'CACHE_DATABASE': '{{ netbox_redis_cache_database }}',
    'DEFAULT_TIMEOUT': '{{ netbox_redis_default_timeout }}',
    'SSL': '{{ netbox_redis_ssl_enabled }}',
}
```
I can create PR for this.

**Comment by lae** [comment]
The SSL key expects a string? A PR would be appreciated if you can.

**Comment by kdhlab** [comment]
The redis configuration syntax changed with 2.7. Per the documentation:

https://netbox.readthedocs.io/en/latest/release-notes/version-2.7/

```
REDIS = {
    'webhooks': {
        'HOST': 'redis.example.com',
        'PORT': 1234,
        'PASSWORD': 'foobar',
        'DATABASE': 0,
        'DEFAULT_TIMEOUT': 300,
        'SSL': False,
    },
    'caching': {
        'HOST': 'localhost',
        'PORT': 6379,
        'PASSWORD': '',
        'DATABASE': 1,
        'DEFAULT_TIMEOUT': 300,
        'SSL': False,
    }
}
```

I am working through making this work in 2.7, if people are still deploying 2.6 we may need two sets of templates?

**Comment by lae** [comment]
For now, please target your changes for NetBox 2.6. The role has yet to be updated to work for 2.7.

**Comment by lae** [comment]
Fixed in #73
